### PR TITLE
ref(committers): Redesign suspected commits

### PR DESCRIPTION
**Pull request by dcramer**
This utilizes the existing commit styling used in releases and expands the list to include all suspects.

### CODE REVIEW

**Comment by dcramer** [comment]
screen shot 2018-02-28 at 3 55 23 pm

File: src/sentry/api/endpoints/event_file_committers.py [hunk 1]
```diff
@@ -89,10 +89,6 @@ def _get_commits_committer(self, commits, author_id):
         committer_commit_list = [
             serialize(commit) for commit in commits if commit.author.id == author_id
         ]
-
```
Comment (dcramer) [inline src/sentry/api/endpoints/event_file_committers.py]: i cant be asked to make a mapping of this -- we can clean up and make this endpoint more efficient later

**Comment by dcramer** [comment]
followup: this should go into EventEntries, as it should go below processing errors

File: src/sentry/static/sentry/app/components/events/eventCause.jsx [hunk 2]
```diff
@@ -149,42 +178,16 @@ export default createReactClass({
...
-        <div style={{marginBottom: 20, fontSize: '0.7em', color: '#999', lineHeight: 1}}>
-          {!!probablyTheCommit.author ? (
-            <strong>{probablyTheCommit.author.name}</strong>
-          ) : (
-            <strong>
-              <em>Unknown Author</em>
-            </strong>
-          )}{' '}
-          committed <TimeSince date={probablyTheCommit.dateCreated} />
+          <h3>Suspect Commits ({commitsWithAge.length})</h3>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/eventCause.jsx]: translate

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/components/events/eventCause.jsx (hunk 2 above)
Comment (dcramer) [reply src/sentry/static/sentry/app/components/events/eventCause.jsx]: done -- also removed a bunch of dead code that i accidentally left in

**Comment by dcramer** [comment]
I moved this below errors, and moved errors above feedback

**Comment by dcramer** [comment]
Disabling this feature on shared issues.

**Pull request merged**